core/src/main/resources/reference.conf

```diff
@@ -4,13 +4,16 @@ kanaloa {
   default-dispatcher {
     workTimeout = 1m
 
+    # how often dispather makes ajustment accordingly
```

Comment:
This value set the `sampleRate` in the `MetricsCollectorSettings` which is hidden from public. It's also the default value how often AutoScaling tries an action. It's also going to set a `updateInterval` in the incoming `TrafficRegulator` (how often it adjust drop rates) . So it's a global `updateInterval` for several components.